modules/localgov_media/config/optional/core.entity_view_display.media.image.localgov_event_thumbnail.yml

```diff
@@ -0,0 +1,32 @@
+langcode: en
```

Comment:
I think this file needs to be removed from this PR and added to localgov_events repo instead.

Reply:
Thanks @markconroy  

Does that mean I should:

i) re-submit this pull request without this file?
ii) create a new pull request with this file in web/modules/contrib/localgov_events/config/optional ?

Reply:
This file has been removed from this pr with the latest commit.

Reply:
The file in the localgov_events repo is already set to "Hidden" so I have closed the issue, deleted the branch and closed the pull request I created for it in the localgov_events repo.